src/schema/users.ts

```diff
@@ -1759,6 +1765,18 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         });
       }
 
+      await sendEmail({
+        transactional_message_id:
+          CioTransactionalMessageTemplateId.VerifyCompany,
```

Comment:
We don't need sender/subject for email?

Reply:
I think we need `send_to_unsubscribed: true` so verification is sent to unsubscribed users as well.

Reply:
Oh, yeah. Added